demo/runners/support/agent.py

```diff
@@ -732,7 +731,7 @@ async def register_or_switch_wallet(
         public_did=False,
         webhook_port: int = None,
         mediator_agent=None,
-        cred_type: str = CRED_FORMAT_INDY,
+        cred_type: str = CRED_FORMAT_INDY or CRED_FORMAT_VC_DI,
```

Comment:
Same as previous. Isn't this a bug?

Reply:
I think @dhh1128 is correct this is a bug

Reply:
it is a confusing default and a no-op change.  lets leave it as it was, with INDY being the default (which it still is, we are just confusing people)